charts/trino/templates/_helpers.tpl

```diff
@@ -125,17 +125,33 @@ Code is inspired from bitnami/common
 {{- end -}}
 
 {{/*
-Create the name of the file auth secret to use
+Create the name of the file password auth secret to use
 */}}
-{{- define "trino.fileAuthSecretName" -}}
+{{- define "trino.passwordSecretName" -}}
 {{- if and .Values.auth .Values.auth.passwordAuthSecret }}
 {{- .Values.auth.passwordAuthSecret | trunc 63 | trimSuffix "-" }}
 {{- else }}
 {{- $name := default .Chart.Name .Values.nameOverride }}
 {{- if hasPrefix .Release.Name $name }}
-{{- printf "%s-%s" $name "file-authentication" | trunc 63 | trimSuffix "-" }}
+{{- printf "%s-%s" $name "file-password-authentication" | trunc 63 | trimSuffix "-" }}
```

Comment:
`file-password-authentication` is a weird name. The original name meant file-based authentication. How about `password-file`?